apk-size/apk-size.gradle

```diff
@@ -24,6 +24,10 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.dorongold.task-tree" version "1.5"
```

Comment:
What is this plugin for?

Reply:
https://github.com/dorongold/gradle-task-tree

This is used to print out a tree of task dependencies. It is added to the `apk-size` project only. The main project will not be affected.